docs/products/flink/howto/real_time_alerting_solution_cli.rst

```diff
@@ -0,0 +1,428 @@
+
+#. Run the following command to create a data pipeline job named ``simple_filter_pg``:
+
+   ::
+
+      avn service flink job create demo-flink simple_filter_pg            \
+          --table-ids CPU_IN_ID CPU_OUT_FILTER_PG_ID SOURCE_THRESHOLDS_ID \
+          --statement "JOB_SQL"
+
+   Replace the values for ``--table-ids`` with the IDs for the ``CPU_IN``, ``CPU_OUT_FILTER_PG``, and ``SOURCE_THRESHOLDS`` tables, and ``JOB_SQL`` with the following:
+
+   .. literalinclude:: /code/products/flink/alerting_solution_sql.md
+      :lines: 69-77
+      :language: sql
+
+
+Create an aggregated data pipeline with Kafka and PostgreSQL
+------------------------------------------------------------
+
+This setup highlights the instances where the average CPU load over a :doc:`windowed interval </docs/products/flink/concepts/windows>` exceeds the threshold and stores the results in PostgreSQL.
+
+.. mermaid::
+
+    graph LR;
+
+        id1(Kafka source)-- timestamped metrics -->id3(Flink job);
+		id2(PosgreSQL source)-- host-specific thresholds -->id3;
+        id3-- high 30-second average CPU -->id4(PostgreSQL sink);
+
+
+1. In the Aiven CLI, run the following command to connect to the ``demo-postgresql`` service:
+   
+   ::
+	  
+      avn service cli demo-postgresql --project PROJECT_NAME
+   
+#. Enter the following command to set up the PostgreSQL table for storing the results:
+   
+   .. literalinclude:: /code/products/flink/alerting_solution_sql.md
+      :lines: 80-83
+      :language: sql
+   
+#. Run the following command to create a PostgreSQL table named ``CPU_OUT_AGG_PG``:
+
+   ::
+
+      avn service flink table create demo-flink POSTGRESQL_INTEGRATION_ID  \
+          --table-name CPU_OUT_AGG_PG                                      \
+          --jdbc-table cpu_load_stats_agg_pg                               \
+          --schema-sql "TABLE_SQL"
+
+   Replace ``POSTGRESQL_INTEGRATION_ID`` with the ID for your ``demo-postgresql`` service integration and replace ``TABLE_SQL`` with the following:
+
+   .. literalinclude:: /code/products/flink/alerting_solution_sql.md
+      :lines: 86-88
+      :language: sql
+
+#. Run the following command to list the tables for the ``demo-flink`` service and get the IDs for the ``CPU_IN``, ``CPU_OUT_AGG_PG``, and ``SOURCE_THRESHOLDS`` tables:
+
+   ::
+
+      avn service flink table list demo-flink
+
+#. Run the following command to create a data pipeline job named ``simple_filter_pg_agg``:
+
+   ::
+
+      avn service flink job create demo-flink simple_filter_pg_agg     \
+          --table-ids CPU_IN_ID CPU_OUT_AGG_PG_ID SOURCE_THRESHOLDS_ID \
+          --statement "JOB_SQL"
+
+   Replace the values for ``--table-ids`` with the IDs for the ``CPU_IN``, ``CPU_OUT_AGG_PG``, and ``SOURCE_THRESHOLDS`` tables, and ``JOB_SQL`` with the following:
+
+   .. literalinclude:: /code/products/flink/alerting_solution_sql.md
+      :lines: 91-124
+      :language: sql
+
+
```

Comment:
What's the punchline? Did I make something? What was it, what problem did I solve? I'd love to see a bit of a recap of the steps and maybe how they could apply to other use cases. We can also add some further reading here if there are links people might like to integrate next.

Reply:
Expected results added to the end of each section.